rdmo/projects/assets/js/interview/components/main/page/PageHeadFormModal.js

```diff
@@ -22,10 +22,14 @@ const PageHeadFormModal = ({ title, submitLabel, submitColor, show, attribute, r
   const [errors, setErrors] = useState([])
 
   const handleSubmit = () => {
-    if (isEmpty(values.text)) {
-      setErrors({ text: true })
+    if (isNil(attribute)) {
```

Comment:
This is a fix for a different bug, which occurs if Pages don't have an attribute (`#1`, `#2`, ...)